src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public String getFig() {
+        return "circle";
+    }

```

Comment:
getFig() - не информативное название метода. 
```suggestion
    public String getFigureType() {
        return getClass().getSimpleName();
    }
```

И мы можем написать эту реализацию в классе Figure и не переопределять ее в классах наследниках